RELEASE_NOTES.md

```diff
@@ -3,8 +3,33 @@ Release notes
 
 ### Next
 
-* Included support for compilation on Java 9 and 10. Compilation on Java 9 or 10 will produce a multi-release JAR that will run on Java 8, 9, and 10. This release was compiled on Java 8 and does not contain a multi-release JAR.
-* Changed metadata health check names to use the URI of the metadata they are trying to resolve
+#### Connect to Verify using only the VSP
+
+This release adds the ability to connect to GOV.UK Verify using only the Verify Service Provider (VSP). This means services can connect without needing to host a [Matching Service Adapter (MSA)](https://github.com/alphagov/verify-matching-service-adapter). This makes it easier to connect to Verify and reduces maintenance tasks once connected.
+
+When used alone, the VSP provides MSA functionality. It can:
+* implement security features to the same security level as the MSA
+* handle European identities
+* do Personal Identifier (PID) hashing
+
+If you are already connected to GOV.UK Verify and are running an MSA, you can upgrade to VSP 2.0.0 without needing to do any configuration changes.
+
+If you want to switch off your Matching Service Adapter and only use the Verify Service Provider to connect to Verify, [contact the GOV.UK Verify Team](https://www.verify.service.gov.uk/support/).
+
+For more information on connecting to GOV.UK Verify using only the VSP, see the [technical documentation](https://www.docs.verify.service.gov.uk).
+
+#### Tutorial for the new VSP
+
+We moved the [VSP tutorial](https://www.docs.verify.service.gov.uk/get-started-with-vsp/#get-started-with-the-verify-service-provider) to the new technical documentation website and updated it to help you connect to GOV.UK verify using the VSP.
+
+#### Support for compilation on Java 9 and 10
+
+Compilation on Java 9 or 10 will produce a multi-release JAR that will run on Java 8, 9, and 10. This release was compiled on Java 8 and does not contain a multi-release JAR.
+
+#### Metadata health check changes
+
+Changed **HOW?** metadata health check names to use the URI of the metadata they are trying to resolve
```

Comment:
Someone needs to dig into the history and either find out what was done, or find out how did it and ask them.  I'd prefer not to be sucked into this while I'm on the yak desk, so please ask another dev on the team.

Reply:
Spoke to Andy and it's all sorted now 👍